src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
+import cn from 'classnames';
+import { peopleFromServer } from '../data/people';
+import { Person } from '../types/Person';
+
+type Props = {
+  setSelectedPerson: (person: Person) => void;
+};
+
+const DELAY = 1000;
+
+export const DropdownMenu: React.FC<Props> = ({ setSelectedPerson }) => {
+  const [showUserList, setShowUserList] = useState(false);
+  const [query, setQuery] = useState('');
+  const [aplliedQuery, setAplliedQuery] = useState('');
+
+  const filteredPeople = useMemo(
+    () => peopleFromServer.filter((person: Person) => person.name
+    // eslint-disable-next-line @typescript-eslint/indent
+    .toLowerCase().includes(aplliedQuery.toLowerCase().trim())), [aplliedQuery],
```

Comment:
do not disable eslint if it is unnessary
```suggestion
      .toLowerCase()
      .includes(aplliedQuery.toLowerCase().trim())), [aplliedQuery],
```